content/html/concepts/favicon/favicon.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: 'Favicon'
+Description: "A favicon is a small icon that is displayed in the browser's address bar and tab title for a website."
```

Comment:
```suggestion
Description: "A favicon is an icon that is displayed in the browser's address bar and tab title for a website."
```

Reply:
This is inferred ^^

Reply:
Thank you for the input :) I'm going to try the "commit" button here and see how it works. XD

Reply:
Correct me if I'm wrong, but that button just made the changes for me and made a commit, and just added you as a 'co-author'?